From 1d911520305f705aa10e407f4855d2286676a5b8 Mon Sep 17 00:00:00 2001
From: Zbigniew Bodek <zbb@semihalf.com>
Date: Mon, 30 Jul 2012 11:03:33 +0200
Subject: [PATCH 0403/1825] Neta: Workaround for compilation errors
 considering mv_eth_tool.c

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 6ab06df35ddf3274fbc84528376957efb84f64f9

- Workaround for compilation errors. There were some changes in network stack
  between 3.2 and 3.5 kernels, that set_tx_csum, set_tsc, set_rxfh_indir,
  set_rx_ntuple etc. and corresponding get_ ops became obsolete or changed.
  This should be reconsidered and adjusted properly.

Change-Id: I21bda2cc99459ba1b674019569ffd2eaec815c60
Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 .../mv_drivers_lsp/mv_neta/net_dev/mv_eth_tool.c   |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/plat-armada/mv_drivers_lsp/mv_neta/net_dev/mv_eth_tool.c b/arch/arm/plat-armada/mv_drivers_lsp/mv_neta/net_dev/mv_eth_tool.c
index 857c782..67d0a5a 100644
--- a/arch/arm/plat-armada/mv_drivers_lsp/mv_neta/net_dev/mv_eth_tool.c
+++ b/arch/arm/plat-armada/mv_drivers_lsp/mv_neta/net_dev/mv_eth_tool.c
@@ -922,9 +922,9 @@ const struct ethtool_ops mv_eth_tool_ops = {
 	.get_stats_count			= mv_eth_tool_get_stats_count,
 #endif
 	.get_ethtool_stats			= mv_eth_tool_get_ethtool_stats,
-	.get_rxfh_indir				= mv_eth_tool_get_rxfh_indir,
-	.set_rxfh_indir				= mv_eth_tool_set_rxfh_indir,
-	.get_rxnfc                  = mv_eth_tool_get_rxnfc,
-	.set_rx_ntuple				= mv_eth_tool_set_rx_ntuple,
+//	.get_rxfh_indir				= mv_eth_tool_get_rxfh_indir,
+//	.set_rxfh_indir				= mv_eth_tool_set_rxfh_indir,
+//	.get_rxnfc                  = mv_eth_tool_get_rxnfc,
+//	.set_rx_ntuple				= mv_eth_tool_set_rx_ntuple,
 };
 
-- 
1.7.5.4

